pkgs/tools/archivers/ctrtool/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "ctrtool";
+  version = "0.15";
+
+  src = fetchFromGitHub {
+    owner  = "profi200";
+    repo   = "Project_CTR";
+    rev    = "${version}";
```

Comment:
Sorry missed this first time around. Should be `rev = version;`, no need for quoting.